server/protocols/jmap-rfc-8621/doc/specs/spec/mail/searchsnippet.mdown

```diff
@@ -0,0 +1,94 @@
+# Search Snippets
+
```

Comment:
Not implemented (and likely won't)

Reply:
> and likely won't

curious about why

Reply:
No you are right, this is an unjustificated claim.